src/chrome/content/rules/Bundesbank.de.xml

```diff
@@ -0,0 +1,54 @@
+<!--
+	Invalid certificate:
+		(www.)?glaeubiger-id
+		www.immobilien
+		onlinebanking
+		(www.)?target2
+
+	Timeout:
+		ebics-t1
+		extranet-t
+		newsletter
+		s504pmgwz1
+		s999mgwxx001a
+		s999mgwxx001p
+		s999mgwxx002a
+		s999mgwxx002p
+		smv
```

Comment:
I get a mismatch, but it works on port 8443

Reply:
I still get timeouts on both ports, 443 and 8443.